### PR TITLE
Update docker-compose API port

### PR DESCRIPTION
**Pull request by johnelliott**
Its possible that I didn't initialize my env and that this should have worked without this change. The api was starting on port 5000 inside docker so I updated the port so the 8080 I saw elsewhere in the app would be the API when I did a curl from my shell.

### CODE REVIEW

File: components/sickdb-api/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,8 @@ MAINTAINER Jeremy Krinsley <dev@globally.ltd>
 
 # Install package requirements
 
-RUN apt update && apt install -y \
+RUN rm -rf /var/lib/apt/lists/* && apt update && apt-get install -y \
```
Comment (johnelliott) [inline components/sickdb-api/Dockerfile]: This may be a one-time thing I needed to bootstrap

Comment (abelsonlive) [reply components/sickdb-api/Dockerfile]: what is going on here?

Comment (johnelliott) [reply components/sickdb-api/Dockerfile]: I was getting an error
```E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/g/gcc-5/libcc1-0_5.4.0-6ubuntu1~16.04.6_amd64.deb  Hash Sum mismatch

ERROR: Service 'api' failed to build: The command '/bin/sh -c apt update && apt install -y 	--fix-missing 	curl 	libldap2-dev 	libsasl2-dev 	postgresql 	python3.5 	python3-pip 	python3-psycopg2 	uwsgi 	uwsgi-plugin-python3 	git 	&& rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
``` 

when I was doing the initial build/run of docker. I googled a bit and found that I needed to do this before apt would be happy: https://askubuntu.com/questions/41605/trouble-downloading-packages-list-due-to-a-hash-sum-mismatch-error/

There's a chance this isn't needed. It seems dubious being before and after the install.

**Comment by abelsonlive** [comment]
yeah this is still weird to me cause i changed the default port on the api but docker is still starting it on 8080

**Pull request closed without merging**